english/make-friends/make-friends.md

```diff
@@ -13,3 +13,5 @@ Do you:
 [Attempt to speak the marshmallowish language](marshmallowish/marshmallowish.md)
 
 [Have a cup of coffee?](../coffee/coffee.md)
+
+[Ask the marshmellow man, "What's it all for?"](../question/question.md)
```

Comment:
Since the `question` directory is within the same directory as this file, the `..` is not necessary. The link should be just `question/question.md`